shell/platform/common/cpp/text_input_model.cc

```diff
@@ -90,38 +87,34 @@ void TextInputModel::AddText(const std::string& text) {
 }
 
 bool TextInputModel::Backspace() {
-  // If there's a selection, delete it.
   if (DeleteSelected()) {
     return true;
   }
-  // There's no selection; delete the preceding codepoint.
```

Comment:
I think the reference to deleting the codepoint is useful context for the code that follows, and should stay.

Reply:
Good point -- done.